src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  people: Person[];
+  slug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, slug }) => {
+  const isSelected = (person: Person) => person.slug === slug;
+
+  const personsMotherLink = (person: Person) => {
+    const mother = people.find(p => p.name === person.motherName);
+
+    if (mother) {
+      return (
+        <PersonLink person={mother} />
+      );
+    }
+
+    return person.motherName
+      ? person.motherName
+      : '-';
+  };
+
+  const personsFatherLink = (person: Person) => {
+    const father = people.find(p => p.name === person.fatherName);
+
+    if (father) {
+      return (
+        <PersonLink person={father} />
+      );
+    }
+
+    return person.fatherName
+      ? person.fatherName
+      : '-';
+  };
```

Comment:
you can create one function like getPersonLinkByName() and just pass a name you are looking for 
```
getPersonLinkByName(person.fatherName) 
getPersonLinkByName(person.motherName) 
```

Reply:
good catch! fixed it, thanks